client.go

```diff
@@ -95,7 +99,8 @@ func (c *Client) Connect() (err error) {
 		c.Port = 22
 	}
 	config := &ssh.ClientConfig{
-		User: c.User,
+		User:            c.User,
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
```

Comment:
I would rather not make this the default (but maybe some optional config). See  https://github.com/golang/crypto/blob/master/ssh/client.go#L229 (It should not be used for production code.)